doc/user-guide/user_guide.md

```diff
@@ -1,71 +1,41 @@
-[Redshift][redshift-dialect-doc]    |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[SAP HANA][sap-hana-dialect-doc]    |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[SQL Server][sql-server-dialect-doc]|  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[Sybase ASE][sybase-dialect-doc]    |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[Teradata][teradata-dialect-doc]    |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-Generic                             |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |
-
-
-# JDBC Adapter for Virtual Schemas
-
-## Overview
 
 The JDBC adapter for virtual schemas allows you to connect to JDBC data sources like Hive, Oracle, Teradata, Exasol or any other data source supporting JDBC.
 It uses the well proven ```IMPORT FROM JDBC``` Exasol statement behind the scenes to obtain the requested data, when running a query on a virtual table. 
 The JDBC adapter also serves as the reference adapter for the Exasol virtual schema framework.
 
-You can find supported dialects in [the list of supported dialects](#list-of-supported-dialects).
+You can find supported dialects in [the list of supported dialects](dialects.md).
 
 The so called `GENERIC` dialect is designed to work with any JDBC driver. It derives the SQL dialect from the JDBC driver metadata. 
 However, it does not support any capabilities and might fail if the data source has special syntax or data types, so it should only be used for evaluation purposes.
 
 If you are interested in an introduction to virtual schemas please refer to the [Exasol Documentation Portal](https://docs.exasol.com/home.htm). 
 
-## Before you Start
+## Before you Start 
 
 Please note that the syntax for creating adapter scripts is not recognized by all SQL clients. 
-[DBeaver](https://dbeaver.io/) for example. If you encounter such a problem, try a different client.
+See [SQL Client Specifics](sql_clients.md) for details
 
 ## Getting Started
 
-This page contains common information applicable to all the dialects. You can also check more [detailed guides for each dialect](#list-of-supported-dialects) in the dialects' documentation.
+This page contains common information applicable to all the dialects. You can also check more [detailed guides for each dialect](dialects.md) in the dialects' documentation.
+
+The steps for creating virtual schema are:
+
+* Deploying JDBC Driver Files
+* Install the adapter script
```

Comment:
Use nouns everywhere or rephrase the sentence and use verbs everywhere. It's a mix with strange grammar now